lib/slices/tree.rb

```diff
@@ -39,7 +39,7 @@ module Tree
       field :position, type: Integer
       field :show_in_nav, type: Boolean, default: false
 
-      scope :minimal, ->{ only(%w(_type page_id path external_url name has_content)) }
+      scope :minimal, ->{ only(%w(_id _type page_id path external_url name has_content)) }
```

Comment:
It's fixed in Mongoid 5 https://github.com/mongodb/mongoid/commit/d873923910f5a4a419bc5a4fd4e9c7adc317e29b